### DISCUSSION TITLE
Getting ERROR CODE: InvalidAuthenticationTokenTenant

### DISCUSSION THREAD

**Discussion by DaveLHX** [Azure/azure-dev#1440] [Q&A] [answered]
I have a tenant setup with my MSDN subscription. If I run `azd login` then `azd init` I dont see my subscription anymore and If I try to pass it I get the following error.
```
ERROR: loading environment: initializing environment: prompting for location: listing locations: failed retrieving Azure location for account 'MY_SUBSCRIPTION_ID': failed getting next page of locations: GET https://management.azure.com/subscriptions/MY_SUBSCRIPTION_ID/locations
--------------------------------------------------------------------------------
RESPONSE 401: 401 Unauthorized
ERROR CODE: InvalidAuthenticationTokenTenant
--------------------------------------------------------------------------------
{
  "error": {
    "code": "InvalidAuthenticationTokenTenant",
    "message": "The access token is from the wrong issuer 'https://sts.windows.net/NOT_MY_TENANT_ID/'. It must match the tenant 'https://sts.windows.net/MY_TENANT_ID/' associated with this subscription. Please use the authority (URL) 'https://login.windows.net/MY_TENANT_ID' to get the token. Note, if the subscription is transferred to another tenant there is no impact to the services, but information about new tenant could take time to propagate (up to an hour). If you just transferred your subscription and see this error message, please try back later."
  }
}
--------------------------------------------------------------------------------
```
I dont know if its related but in November I was able to run `azd init` and would get my subscription. The subscription got disabled because I was busting the limits but it was reactivated yesterday.
I double checked that `azd login` does in fact connect me to the correct tenant.  Also the contents of the azureProfile.json seem to ok from what I understand.

running this version `azd version 0.5.0-beta.3 (commit 1c23678aade587d5b9685a256a3c1d2d7f4b42e0)`

**Comment by DaveLHX** [comment, marked as answer]
So I read the version history and a recently solved bug seems similar but not exactly the same but the work around worked.
https://github.com/Azure/azure-dev/issues/1366#issuecomment-1380842081

**Reply by rajeshkamal5050** [reply]
@DaveLHX thanks for starting a discussion on this.

We have a tracking issue for the same. Team is working on fixing it. Should be available soon.
- https://github.com/Azure/azure-dev/issues/1409

Please use the workaround you have referenced above.
